src/nunit.analyzers/SimplifyValues/SimplifyValuesCodeFix.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using NUnit.Analyzers.Constants;
+
+namespace NUnit.Analyzers.SimplifyValues;
+
+[ExportCodeFixProvider(LanguageNames.CSharp)]
+[Shared]
+public class SimplifyValuesCodeFix : CodeFixProvider
+{
+    internal const string SimplifyValuesTitle = "Simplify the Values attribute";
+
+    public override ImmutableArray<string> FixableDiagnosticIds
+        => ImmutableArray.Create(AnalyzerIdentifiers.SimplifyValues);
```

Comment:
```suggestion
    public override ImmutableArray<string> FixableDiagnosticIds { get; }
        = ImmutableArray.Create(AnalyzerIdentifiers.SimplifyValues);
```

The original version would create a new instance every time it is called.
Yes, I know not all the Analyzer use this (yet)